dexios/src/subcommands/header_key.rs

```diff
@@ -140,7 +143,312 @@ pub fn change_key(input: &str, key_old: &Key, key_new: &Key) -> Result<()> {
             success!("Key successfully updated for {}", input);
         }
         HeaderVersion::V5 => {
-            todo!()
+            let mut keyslots = header.keyslots.clone().unwrap();
+
+            if keyslots.len() < 1 {
```

Comment:
I suggest to use the following code:

```
let keyslot = keyslots.iter().next().ok_or_else(|| anyhow::anyhow!("No keyslots found, so we cannot continue."))?;
```

and then `keyslot` instead of `keyslots[0]`

Reply:
I like this change! I'll implement it now :)